datasetinsights/io/tracker/factory.py

```diff
@@ -55,23 +56,19 @@ def create(config=None, tracker_type=None):
             raise InvalidTrackerError
 
     @staticmethod
-    def _mlflow_tracker_instance(mlflow_config):
+    def _mlflow_tracker_instance(**kwarg):
```

Comment:
[nit] Use `**kwargs` instead. This is a variable commonly used in python packages.